internal/resolvers/imgur/resolver.go

```diff
@@ -1,8 +1,10 @@
+//go:generate mockgen -destination ../../mocks/mock_imgurClient.go -package=mocks . ImgurClient
+
```

Comment:
Could you maybe add a little comment explaining what does this command do? Also, shouldn't it be `go generate` instead?

Reply:
A comment has been made in the file now, copying the comment here for clarification:
// The above comment will make it so that when `go generate` is called, the command after go:generate is called in this files PWD.
// The mockgen command itself generates a mock for the ImgurClient interface in this file and stores it in the internal/mocks/ package